src/main/java/net/mcreator/ui/modgui/BlockGUI.java

```diff
@@ -168,8 +168,9 @@
 
 	private final MCItemHolder customDrop = new MCItemHolder(mcreator, ElementUtil::loadBlocksAndItems);
 
-	private final JSpinner minGenerateHeight = new JSpinner(new SpinnerNumberModel(0, 0, 256, 1));
-	private final JSpinner maxGenerateHeight = new JSpinner(new SpinnerNumberModel(64, 0, 256, 1));
+	private final JComboBox<String> generationShape = new JComboBox<>(new String[] { "UNIFORM", "TRIANGLE" });
+	private final JSpinner minGenerateHeight = new JSpinner(new SpinnerNumberModel(0, -2032, 2016, 1));

```

Comment:
The page attached in the PR description states that this height
>Should be between -2032 and 2031 and be a multiple of 16

So the step of these spinners should be 16, not 1.

Reply:
16 is for the heigh of the dimension only. It doesn't apply for blocks.

Reply:
Oh, sorry.